autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -327,16 +330,16 @@ def add_model(self, model: AbstractModel, stack_name: str, level: int):
         if self.low_memory:
             del model
 
-    def train_single_full(self, X_train, y_train, X_val, y_val, model: AbstractModel, feature_prune=False,
-                          hyperparameter_tune=True, stack_name='core', kfolds=None, k_fold_start=0, k_fold_end=None, n_repeats=None, n_repeat_start=0, level=0, time_limit=None):
+    def _train_single_full(self, X_train, y_train, X_val, y_val, model: AbstractModel, feature_prune=False,
```

Comment:
documentation should describe what this method does beyond training & saving

Reply:
added